README.md

```diff
@@ -23,8 +23,7 @@ A puppet module for the Unbound caching resolver.
 
 ## Requirements
 
-To use this module, you must be running at least Puppet 4.4.  If you are on an
-older version of Puppet, please use a 1.x version of this module.
+To use this module, you must be running at least Puppet 6 and unbound 1.6.6
 
 To use this module, you must install the `puppetlabs/concat` and
```

Comment:
And this should IMHO refer to metadata.json too. Too easy to go out of sync.

Reply:
ack sgtm will update